modules/ezcontent_node/modules/ezcontent_landing_page/ezcontent_landing_page.install

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Install file for ezcontent_landing_page.
+ */
+
+/**
+ * Remove paragraph content inline block option from layout.
+ */
+function ezcontent_landing_page_update_8001() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('ezcontent_landing_page', 'ezcontent_landing_page_update_8001');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
```

Comment:
We could avoid this as per my comment: https://github.com/srijanone/ezcontent/pull/332/files#r580805147